frontend/src/journal/utils/textEditor.ts

```diff
@@ -0,0 +1,11 @@
+export const modules = {
+  toolbar: [
+    ["bold", "italic", "underline", "strike"],
+    ["link", "image", "video"],
```

Comment:
Does the image and video upload work? 🤔

Reply:
It doesnt so I just remove it mb in the future